src/views/PurchaseOrder.vue

```diff
@@ -134,10 +134,14 @@ export default defineComponent({
           orderItem.facilityId = item[this.facilityField]
           return orderItem
         })
-        this.store.dispatch('order/updatedOrderList', this.orderItemsList);
-        this.router.push({
-          name:'PurchaseOrderDetail'
-        })
+        if(this.orderIdField && this.productSkuField && this.dateField && this.quantityField && this.facilityField && this.orderItemsList.length && this.content){
+          this.store.dispatch('order/updatedOrderList', this.orderItemsList);
+          this.router.push({
+            name:'PurchaseOrderDetail'
+          })
+        } else {
+          showToast(translate("Something went wrong"));
```

Comment:
if this is an error message just for this condition, we should specifically say:

"Please upload a purchase order to continue"

Reply:
Updated the message for case when file is not uploaded.